app/models/test_result.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class TestResult < ApplicationRecord
+  belongs_to :profile
```

Comment:
Technically, this should be `has_one :profile` (profile_id should be stored on the TestResult) and should be optional

![image](https://user-images.githubusercontent.com/761923/70163598-bfa9d680-168d-11ea-8d93-75bdfc541b5c.png)